templates/nextjs/package.json

```
@@ -9,7 +9,9 @@
     "build": "pnpm run xprebuild && next build",
     "start": "next start",
     "lint": "next lint",
-    "test": "vitest"
+    "test": "run-s test:*",
```

I believe we removed all instances of wildcard `*` running for a reason.

```suggestion
    "test": "run-s test:unit test:e2e",
```